### PR TITLE
Implement data export for e2e drift testing

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

File: test/driftexport/drift_export_test.go [hunk 1]
```diff
@@ -0,0 +1,733 @@
...
+	q := psql.Select(fmt.Sprintf("to_jsonb(%s)", table.Name)).From(table.Name)
+	q, err := createFilter(parent, table, resource, q)
+	if err != nil {
+		return nil, err
+	}
+	sql, args, err := q.ToSql()
+	if err != nil {
+		return nil, err
+	}
+	rows, err := conn.Query(ctx, sql, args...)
```
Comment (disq) [inline test/driftexport/drift_export_test.go:609]: `defer rows.Close()`

File: test/driftexport/drift_export_test.go [hunk 2]
```diff
@@ -0,0 +1,733 @@
...
+		}
+	}
+	return b
+}
+
+func makeIDBuilder(expr string, table *schema.Table) func([]map[string]interface{}) string {
+	if expr == "" {
+		keys := make([]string, 0, len(table.Options.PrimaryKeys))
+		for _, k := range table.Options.PrimaryKeys {
+			if k != "account_id" && k != "region" && k != "cq_id" && !strings.HasSuffix(k, "_cq_id") {
```
Comment (disq) [inline test/driftexport/drift_export_test.go:673]: You could use the `column.Meta().Resolver.Name` ...

**Review by disq — APPROVED** [review]

File: test/driftexport/drift_export_test.go (hunk 1 above)
Comment (irmatov) [reply test/driftexport/drift_export_test.go:609]: Good catch, thanks

File: test/driftexport/drift_export_test.go (hunk 2 above)
Comment (irmatov) [reply test/driftexport/drift_export_test.go:673]: Well, yeah, I've seen how it's done in drift code, but this was quicker.

Shamefully adopted some of the code from there. Thanks.

**Pull request closed without merging**